### ISSUE TITLE
Unable to install packages when user name contains non-ASCII characters

### ISSUE THREAD

**Issue by Callidior** [microsoft/winget-cli#181]
# Brief description of your issue

My windows user name is "Björn", and hence contains a non-ASCII character. When I try to install any package using `winget`, I receive an error message similar to the following one:

> "C:\Users\BjÃ¶rn\AppData\Local\Packages\Microsoft.DesktopAppInstaller_8wekyb3d8bbwe\TempState\WinGet\gimp.gimp.2.10.18.exe" konnte nicht gefunden werden. Stellen Sie sicher, dass Sie den Namen richtig eingegeben haben und wiederholen Sie den Vorgang.

The specified path indeed does not exist. Replacing "BjÃ¶rn" with my correct username, "Björn", though, leads to the correct file downloaded by `winget`.

# Steps to reproduce

1. Log on with a Windows username containing non-ASCII characters.
2. Open `cmd`.
3. Run: `winget install gimp.gimp -e`

# Expected behavior

`winget` should download the installer and execute it.

# Actual behavior

`winget` downloads the installer, but fails to execute it, because it uses an invalid path for running the downloaded file. This leads to the following error message in a message box window:

> "C:\Users\BjÃ¶rn\AppData\Local\Packages\Microsoft.DesktopAppInstaller_8wekyb3d8bbwe\TempState\WinGet\gimp.gimp.2.10.18.exe" konnte nicht gefunden werden. Stellen Sie sicher, dass Sie den Namen richtig eingegeben haben und wiederholen Sie den Vorgang.

On the command line, I get this message:

> Installer failed with exit code: 1223

# Environment

```none
[winget --info]
Windows Package Manager v0.1.41331 Vorschau
Copyright (c) Microsoft Corporation. All rights reserved.

Windows: Windows.Desktop v10.0.18363.836
Package: Microsoft.DesktopAppInstaller v1.0.41331.0
```

**Comment by tomgron** [comment]
Got this too : 
![image](https://user-images.githubusercontent.com/1743381/83458351-0d4fa200-a46b-11ea-92dd-80b250043d1d.png)

**Comment by KristofKekesi** [comment]
me too (my username is Kristóf)

**Comment by Ed1123** [comment]
Any workaround?

**Comment by yao-msft** [comment]
We plan to fix this in v0.2 milestone. Thanks.

**Comment by yao-msft** [comment]
The fix should have been included with release [v0.2.2521](https://github.com/microsoft/winget-cli/releases/tag/v.0.2.2521-preview). Below is a working snapshot.

![Capture](https://user-images.githubusercontent.com/50888816/94088633-d4df2580-fdc5-11ea-8a57-3dd5f7c8ee58.JPG)

Please feel free to reopen the issue if the fix does not work as expected.
